tests/ci/profiles/rosa-classic.yaml

```diff
@@ -0,0 +1,155 @@
+profiles:
+- as: rosa-sts-advanced
+  version: latest
+  channel_group: candidate
+  region: "us-west-2"
+  cluster:
+    multi_az: true
+    instance_type: "r5.xlarge"
+    hcp: false
```

Comment:
Still, as same as ocm-backend-tests, I do not like the `false` here. From my side, only the enabled feature here is meaningful.